pages/public-sitemap.tsx

```diff
@@ -110,7 +110,7 @@ export async function getStaticProps() {
         name: titleize(pathParts[0]),
       });
     } else if (pathParts.length > 1) {
-      if (path.includes("integrations")) {
+      if (pathParts[0] === "integrations") {
```

Comment:
now that we have `/docs/integrations` and `/integrations`, we need to be more specific here.